### ISSUE TITLE
RuntimeError: cuDNN error: CUDNN_STATUS_INTERNAL_ERROR？

### ISSUE THREAD

**Issue by 18810580125** [layumi/Person_reID_baseline_pytorch#240]
在训练过程中，出现错误：RuntimeError: cuDNN error: CUDNN_STATUS_INTERNAL_ERROR，不知道如何解决

**Comment by layumi** [comment]
It seems to be one error of pytorch. You may refer to this issue 
https://github.com/pytorch/pytorch/issues/16831

**Comment by niceblue88** [comment]
Yes still broken, at least for me. Pytorch 1.9, Cuda 11 and using Titan X in Windows 10. Benchmark = false works no problem reliably. Benchmark = true works for a bit (with double the speed of = false, which is great), but then randomly breaks for no seeming reason with CUDA ERROR. The fact that is works very well for a bit shows it can work very well (with 2x speed up), but clearly there is a bug somewhere that's been in there for at least 2 years now.
